beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -2856,6 +2857,23 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                     "attn_to_finalized_block",
                 );
             }
+            // Failures from `get_attesting_indices` end up here.
+            AttnError::BeaconChainError(BeaconChainError::AttestationValidationError(e)) => {
+                debug!(
+                    %peer_id,
+                    block_root = ?beacon_block_root,
+                    attestation_slot = %failed_att.attestation_data().slot,
+                    failed_att = ?failed_att,
```

Comment:
Will delete this before merging. I just want to see what these attestations from Caplin actually look like.